crates/pop-cli/src/commands/clean.rs

```diff
@@ -1,12 +1,13 @@
 // SPDX-License-Identifier: GPL-3.0
 
-use crate::cli::traits::*;
+use crate::{style::style, cli::traits::*};
 use anyhow::Result;
 use clap::{Args, Subcommand};
 use std::{
 	fs::{read_dir, remove_file},
 	path::PathBuf,
 };
+use cliclack::{confirm, outro_cancel};
```

Comment:
Remove this. That's why the test is waiting for confirmation.
Use `self.cli.confirm` and `self.cli.outro_cancel `instead.